charts/external-nodes/templates/runtime-environments-cm.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
rename to runtime-environments-configmap.yaml

Reply:
i think -cm is good enough:

kubectl get cm - so it is formal
all our cf-helm charts uses this naming